### ISSUE TITLE
DAMSM model pre-training doesn't work

### ISSUE THREAD

**Issue by Iceland-Leo** [taoxugit/AttnGAN#59]
When I was training the DAMSM model, the accuracy of the model on the training dataset was very high, but the accuracy on the test dataset was very low. No matter how I modify the parameters, I can't solve this problem.

**Comment by Blue-Clean** [comment]
hi， @Iceland-Leo , what  accuracy you mean?

**Comment by Iceland-Leo** [comment]
> hi， @Iceland-Leo , what accuracy you mean?

I designed a method similar to R-precision to calculate the accuracy of matching pictures and sentences, but the result is very low and easy to overfit .In a word, the pre-training result of DAMSM is not good.

**Comment by Blue-Clean** [comment]
ok. i got it. THE NEW PAPER dm-gan LOOKS NICE. you can try it and impove the performace. I am worried aboout my graduate paper!!!

**Comment by Iceland-Leo** [comment]
> ok. i got it. THE NEW PAPER dm-gan LOOKS NICE. you can try it and impove the performace. I am worried aboout my graduate paper!!!

OK, thank you very much. And wish you good luck!
